app/models/host/managed_extensions.rb

```diff
@@ -32,7 +32,8 @@ def setReboot
     # continue. If flash is implemented for new hosts (http://projects.theforeman.org/issues/10559)
     # we can report the error to the user perhaps.
     true
-  rescue ::Foreman::Exception
+  rescue => e
```

Comment:
Generally I prefer to only handle exceptions that are expected, so that really unexpected things that we should fix and handle in some way are reported to us as soon as they're seen.